lexactivator.go

```diff
@@ -835,9 +835,7 @@ func GetLicenseOrganizationAddress(organizationAddress *OrganizationAddress) int
    organizationAddressJson = ctoGoString(&cOrganizationAddress[0])
    if organizationAddressJson != "" {
       address := []byte(organizationAddressJson)
-      json.Unmarshal(address, &organizationAddress)
-   } else {
-      organizationAddress = &OrganizationAddress{}
+      json.Unmarshal(address, organizationAddress)
    }
```

Comment:
Earlier I was passing **organizationAddress to unmarshall and it was working fine too as explained in the post below:

https://stackoverflow.com/questions/35604356/json-unmarshal-accepts-a-pointer-to-a-pointer
Which I think was unnecessary.